package.json

```
@@ -6,7 +6,7 @@
   "keywords": [],
   "author": "Surma <surma@surma.dev>",
   "license": "Apache-2.0",
-  "dependencies": {
+  "peerDependencies": {
     "assemblyscript": "^0.9.4"
```

```suggestion
    "assemblyscript": "*"
```

With major=0 version, every minor version is considered a breaking change in semver, so I am wondering if I should just allow any version for now? WDYT?

---

Yeah you right, we need to use `"assemblyscript": ">=0.9.4"`, or `"*"` if it works.